onboard/catkin_ws/src/controls/scripts/test_state_publisher.py

```diff
@@ -3,7 +3,9 @@
 import rospy
 from geometry_msgs.msg import Pose, Twist
 from nav_msgs.msg import Odometry
```

Comment:
Master branch version of test_state_publisher has a lot of changes we should probably keep, so don't merge this part

Reply:
Don't merge